tests/test_rhp_wrappers.py

```diff
@@ -0,0 +1,428 @@
+"""
+This Python 3.11 code tests the ``rhp_wrappers`` module.
+Beware, these tests cover only some functions and only some scenarios.
+Keep adding tests!
+"""
+
+# *****************************************************************************
+#       Copyright (C) 2023 Nicoletta De Maio <demaion@landcareresearch.co.nz>
+#
+#  Distributed under the terms of the GNU Lesser General Public License (LGPL)
+#                  http://www.gnu.org/licenses/
+# *****************************************************************************
+
+# Import standard modules
+import unittest
+
+# Import my modules
+import rhealpixdggs.rhp_wrappers as rhpw
+
+# Import helper modules
+import numpy as np
+
+
+class RhpWrappersTestCase(unittest.TestCase):
+    def test_geo_to_rhp(self):
```

Comment:
Is it worth adding test cases for malformed/surprising input, perhaps to assert validation errors? Something like a |longitude| > 180?

Reply:
The underlying function that implements the conversion is `RHEALPixDGGS.cell_from_point`. It does have its own tests in `test_dggs.py` and any additions should probably go in there rather than in the wrapper tests.

Coordinates that don't lie in the dggs will result in None being returned, which is passed up through the wrapper (and documented in the function documentation for the wrapper).

I have vague memories of trying to find a test case with invalid coordinates and failing - it's possible that longitude values wrap around the sphere?